src/main/scala/workflow/graph/SavedStateLoadRule.scala

```diff
@@ -0,0 +1,19 @@
+package workflow.graph
+
+/**
+ * A rule to load any saved state for the nodes w/ prefixes attached
+ */
+object SavedStateLoadRule extends Rule {
+  override def apply(plan: Graph, prefixes: Map[NodeId, Prefix]): (Graph, Map[NodeId, Prefix]) = {
+    val newGraph = prefixes.foldLeft(plan) {
```

Comment:
Please document what this thing is actually doing. 

My understanding is this - for each "interesting" prefix that exists in the pipeline (we know that the input to this function is only a set of prefixes that exist in the pipeline), check to see whether it is in the state map. If it is, replace the prefix with an ExpressionOperator which is either a fitted transformer or a cached rdd - correct?